src/App.js

```diff
@@ -2,28 +2,43 @@ import { useState } from 'react'
 import galleries from './data.json'
 import ListPage from './pages/ListPage'
 import DetailsPage from './pages/DetailsPage'
+import { Route, Switch } from 'react-router-dom'
+import { useHistory } from 'react-router-dom/cjs/react-router-dom.min'
 
 export default function App() {
   const [activePage, setActivePage] = useState('list')
```

Comment:
I guess the error comes from these const. You can delete the useState because u don't use'm anymore.